BUG: fix categorical comparison with missing values (#26504 ) (#26514)
pandas-dev/pandas: doc/source/whatsnew/v0.25.0.rst

^^^^^^^^^^^

- Bug in :func:`DataFrame.at` and :func:`Series.at` that would raise exception if the index was a :class:`CategoricalIndex` (:issue:`20629`)

> - Bug in :func:`_cat_compare_op` that would valuate comparison with None to True (:issue:`26504`)

Reviewer: Our `whatsnew` entries try to describe the bugs from the perspective of user-facing functionality.  In this case, the bug is affecting comparisons between `Categorical` and scalars.  Can you update the entry to describe what's being affected there instead?

Reviewer 2: Thank you for the suggestion! Will do!

^^^^^^^^^^^

- Bug in :func:`DataFrame.at` and :func:`Series.at` that would raise exception if the index was a :class:`CategoricalIndex` (:issue:`20629`)

> - Fixed bug in comparison of ordered `Categorical` that contained missing values with a scalar which sometimes incorrectly resulted in True (:issue:`26504`)

Reviewer 2: Thanks! Done

> - Fixed Bug in :func:`_cat_compare_op` that would evaluate comparison of ordered `Categorical` with missing values with scalar to True sometimes (:issue:`26504`)

Reviewer 3: this needs to be user facing

> - Fixed Bug in :func:`_cat_compare_op` that would evaluate comparison of ordered `Categorical` with missing values with scalar to True sometimes (:issue:`26504`)

Reviewer 4: ```suggestion
- Fixed bug in comparison of ordered `Categorical` that contained missing values with a scalar which sometimes incorrectly resulted in True (:issue:`26504`)
```

(we don't want to mention the `_cat_compare_op`, as that is not relevant for the end user
